dockerfile_parse/parser.py

```diff
@@ -186,6 +193,18 @@ def _rstrip_backslash(l):
             return l
 
         instructions = []
+        # Handle inherited parent envs
+        for key, val in self.parent_env.items():
+            instructions.append(
```

Comment:
I don't think this is right. The ENV instruction does not appear in the Dockerfile we are parsing. The point of the `structure` attribute is to be able to understand the syntax of the Dockerfile.

The only reason we care about inherited environment variables at all is to be able to know correct values for instructions.